CIPs/0001.md

```diff
@@ -0,0 +1,48 @@
+# CIP 1: Validator-run attestation service
+
+- Date: 2019-08-19
+- Author: @nambrot
+
+### Overview
+
+The [lightweight identity protocol](https://docs.celo.org/celo-codebase/protocol/identity) allows users to map (the hash of) their phone numbers to account addresses, greatly improving the user experience of sending value on the Celo platform.
+
+To do so in a decentralized manner, we rely on validators running what we currently call a verification pool to observe attestation requests and fullfill them by sending them via a text message provider like Twilio. However, the service we created expects celo-blockchain to do the decryption and attestation message signing which may not be ideal. Further, the verification pool should be agnostic to the infrastructure it is run on, whereas now it is tightly coupled to Google Cloud Platform products Firebase Cloud Function and Firebase Database.
```

Comment:
Attestation pool?

Reply:
As per usual, always taking naming suggestions. Right now, I'm suggesting to call it attestation service, as the pool part would be happening within #2